src/inspector_socket_server.cc

```diff
@@ -137,6 +137,21 @@ void SendProtocolJson(InspectorSocket* socket) {
   SendHttpResponse(socket, data);
 }
 
+int GetPort(uv_tcp_t* socket, int* err) {
```

Comment:
Would it make more sense to return the error code and pass the port through a function argument (basically the reverse of the current situation)?